site/app/controllers/student/TeamController.php

```diff
@@ -497,6 +497,28 @@ public function setTeamName($gradeable_id): RedirectResponse {
             $_POST['team_name'] = null;
         }
 
+        if(strlen($_POST['team_name']) > 30) {
```

Comment:
I'd suggest turning this into a constant so its located in one place

The [team model](https://github.com/Submitty/Submitty/blob/main/site/app/models/Team.php) would be a good place, you can add a new constant in the class, something named `MAX_TEAM_NAME_LENGTH = 30` then you can you import the file (`use app\models\Team;`) and access this via `Team::MAX_TEAM_NAME_LENGTH` where needed